tests/services/conftest.py

```diff
@@ -58,7 +58,7 @@ def record_community(db, uploader, minimal_record, community, rdm_record_service
     class Record:
         """Test record class."""
 
-        def create_record(self):
+        def create_record(self, uploader=uploader):
```

Comment:
minor: why is this argument needed?

Reply:
Because sometimes we want to create a record with a different user: for my tests, I need a `community curator` being able to create a record and automatically include it in a community, without a review.